src/app/timetable/services/time-table.service.ts

```diff
@@ -1,9 +1,14 @@
 import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
 
 @Injectable({
   providedIn: 'root'
 })
 export class TimeTableService {
 
-  constructor() { }
+  constructor(private http:HttpClient) { }
+
+  getData() {
```

Comment:
Put a public in face of `getData`.
Rename `getData` in something significant. Like `getTimeTableData()`